### PR TITLE
feat: add simplified Chinese support

### PR DESCRIPTION
**Pull request by LuDreamst**

### CODE REVIEW

File: po/zh_CN.po [hunk 1]
```diff
@@ -0,0 +1,39 @@
+#: rhino-pkg:186
+msgid "Are you sure you want to update all packages? (${BGreen}y${NC}/${BRed}N${NC}) "
+msgstr "确定更新全部软件包？ (${BGreen}y${NC}/${BRed}N${NC}) "
+#: rhino-pkg:218
+msgid "Searching Pacstall…"
+msgstr "正在检索Pacstall..."
```
Comment (Elsie19) [inline po/zh_CN.po:6]: Use the … character instead of the triple dots.

Comment (Elsie19) [inline po/zh_CN.po:6]:
```suggestion
msgstr "正在检索Pacstall…"
```

File: po/zh_CN.po [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+#: rhino-pkg:222
+msgid "Searching apt…"
+msgstr "正在检索apt..."
```
Comment (Elsie19) [inline po/zh_CN.po:9]:
```suggestion
msgstr "正在检索apt…"
```

File: po/zh_CN.po [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+#: rhino-pkg:226
+msgid "Searching flatpak…"
+msgstr "正在检索flatpak..."
```
Comment (Elsie19) [inline po/zh_CN.po:12]:
```suggestion
msgstr "正在检索flatpak…"
```

File: po/zh_CN.po [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+#: rhino-pkg:231
+msgid "Searching snap…"
+msgstr "正在检索snap..."
```
Comment (Elsie19) [inline po/zh_CN.po:15]:
```suggestion
msgstr "正在检索snap…"
```

**Review by Elsie19 — CHANGES_REQUESTED** [review]

**Review by Elsie19 — APPROVED** [review]

**Comment by LuDreamst** [comment]
After receiving emails, I thought I would make revisions later in the evening when I was free. I feel sorry I left you with work I was supposed to do. (>﹏<)

File: po/zh_CN.po (hunk 1 above)
Comment (LuDreamst) [reply po/zh_CN.po]: After receiving emails, I thought I would make revisions later in the evening when I was free. I feel sorry I left you with work I was supposed to do. (>﹏<)

**Pull request merged**